.github/workflows/weekly-pulumi-update.yml

```diff
@@ -120,8 +120,7 @@ jobs:
       run: |
         ver=$(cat .pulumi.version)
         msg="Automated upgrade: bump pulumi/pulumi to ${ver}"
-        # See https://github.com/cli/cli/issues/6485#issuecomment-2560935183 for --head workaround
```

Comment:
Checking if this has been fixed upstream now. If not I'll add the temp workaround to ci-mgmt